doc/runtime.md

```diff
@@ -0,0 +1,60 @@
+# Runtime
+
+## Criterios de decisión
+
+Antes de elegir las herramientas que utilizaremos en nuestro proyecto, es importante tener en cuenta ciertos criterios que nos ayudarán a tomar la mejor decisión. Estos criterios son:
+
+- **Estabilidad**: La herramienta se debe de encontrar en fase estable, no podemos depender de herramientas que estén en fase de desarrollo, o que cuente con muchos errores, ya que sino necesitaremos cambiar cada poco tiempo nuestra herramienta.
+
+- **Mantenimiento**: Es conveniente que la herramienta se vaya actualizando, no sea un runtime desactualizado de hace varios años, que no tiene en cuenta las ultimas versiones.
+
+- **Soporte**: La herramienta que usamos debe de ser una herramienta que tenga soporte para el lenguaje que hemos elegido, en mi caso sería TypeScript.
+
+## Runtimes
+
+Un runtime es un software que interpreta código fuente y lo ejecuta. En nuestro proyecto se ha elegido el lenguaje de programación de TypeScript por lo que debemos de selccionar uno de los diferentes runtimes que podemos encontrar en TypeScript. Entre los runtimes para TypeScript se pueden encontrar los siguientes:
+
+- [Node.js](https://nodejs.org/en/)
+- [Deno](https://deno.com/)
+- [Bun](https://bun.sh/)
+
+### Node.js
+
+Node.js es el más popular de todos para TypeScript. Es un runtime multiplataforma y de código abierto que se ejecuta en muchos sistemas operativos diferentes. Node.js utiliza el motor de JavaScript V8 de Google para ejecutar código JavaScript. Que será necesario para poder ejecutar TypeScript. Node.js tiene una gran comunidad que puede ayudar a resolver problemas. 
```

Comment:
Por favor, tratad de ceñiros a lo que sea a) necesario para distinguir una herramienta de otra y b) relevante para vuestra elección. *Todos* los que listas aquí son multiplataforma. *Todos* son de código abierto. Esto no lo estáis escribiendo para rellenar un número fijo de folios, lo estás haciendo para que yo comprenda por qué habéis tomado una u otra decisión. Y para eso *tengo que leérmelo*. Cuantas más cosas que no sean realmente relevantes incluyáis, menos tiempo puedo dedicar a ayudaros a superar el objetivo a ti o a los compañeros.